control-plane-operator/controllers/hostedcontrolplane/storage/envreplace.go

```diff
@@ -1,9 +1,10 @@
 package storage
 
 import (
-	"k8s.io/utils/strings/slices"
 	"strings"
 
+	"k8s.io/utils/strings/slices"
```

Comment:
~Am I mad or is this not necessary? How is this building?~

Ignore me: I see you just re-arranged this. Sigh, it's late.